dogowner/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class Dogowner(models.Model):
+
+    user = models.ForeignKey(User,
+                             on_delete=models.CASCADE,
+                             blank=True,
+                             null=True
+                             )
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=20)
+    dog_name = models.CharField(max_length=50)
+    dog_race = models.CharField(max_length=50)
+    dog_picture_url = models.CharField(max_length=255)
+    dog_age = models.IntegerField()
+    dog_weight = models.FloatField()
+    dog_gender = models.CharField(
+        max_length=2,
+        choices=Gender.choices,
+        default='M',)
+
+    DogOwners = models.Manager()
+
+    def __str__(self):
+        return self.first_name + self.last_name
+
+    @staticmethod
+    def create_dogowner(email, username, password, dog_name,
+                        first_name, last_name, phone_number,
+                        dog_race, dog_picture_url, dog_age,
+                        dog_weight, dog_gender):
+
+        django_user = User.objects.create_user(username=username,
+                                               email=email,
+                                               password=password
+                                               )
+        new_dog_owner = Dogowner(user=django_user,
+                                 first_name=first_name,
+                                 last_name=last_name,
+                                 phone_number=phone_number,
+                                 dog_name=dog_name,
+                                 dog_race=dog_race,
+                                 dog_picture_url=dog_picture_url,
+                                 dog_age=dog_age,
+                                 dog_weight=dog_weight,
+                                 dog_gender=dog_gender
+                                 )
+        new_dog_owner.save()
+        return new_dog_owner
+
+    @staticmethod
+    def get_user(username):
```

Comment:
is this needed?

Reply:
I tried to provide this model with functions that support several actions,
In second thought, it is not nesscary in this point .
If I will see this method usful in the future, I will add it .